src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -56,7 +58,7 @@ public class ErrorListInfoBarControllerTests
         private ConfigurableConfigurationProvider configProvider;
         private ConfigurableStateManager stateManager;
         private Mock<IKnownUIContexts> knownUIContexts;
-        private ILogger logger;
+        private TestLogger logger;
```

Comment:
Miscellaneous: minor test cleanup. Some of the tests were creating their own `TestLogger` instances if they needed to check logged output. This one already existed, so there was no need.